volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -17,6 +23,22 @@
 vollog = logging.getLogger(__name__)
 
 
+class MFTRecord:
+    # TODO: Change to dataclass with (slots=True) if/when we move minimum
+    # Python version up to 3.10
+    __slots__ = ["record_name", "data_count", "offset"]
+
+    def __init__(
+        self,
+        record_name: Optional[str] = None,
```

Comment:
Allowing all of these to be blank enables the use of `DefaultDict` but strikes me that it could lead to a lot of potential junk entries?